spring-integration-xmpp/src/main/java/org/springframework/integration/xmpp/outbound/ChatMessageSendingMessageHandler.java

```diff
@@ -40,8 +47,11 @@
  */
 public class ChatMessageSendingMessageHandler extends AbstractXmppConnectionAwareMessageHandler {
 
+	private static final Pattern xmlPattern = Pattern.compile("<(\\S.*?)>(.*?)</\\1>");
```

Comment:
`[^>]*>[^<]*<` is generally more efficient than `.*?>.*?` - see [here](http://blog.stevenlevithan.com/archives/greedy-lazy-performance).

You also don't need to capture the value, so I would recommend

`"<(\\S[^>]*)>[^<]*</\\1>"`

Perhaps we should also allow leading and trailing spaces (`\\s*`).